src/software/simulated_tests/simulated_test_fixture.py

```diff
@@ -226,6 +231,19 @@ def runner(
             # Check that all always validations are always valid
             validation.check_validation(always_validation_proto_set)
 
+            if not run_till_end:
+                try:
+                    # Check that all eventually validations are eventually valid
+                    validation.check_validation(eventually_validation_proto_set)
+                    self.__stopper()
+                    return
+                except AssertionError as e:
+                    print("THING")
```

Comment:
is there a reason why we print thing here?

Reply:
forgot to remove that lol thanks